app/controllers/samples_controller.rb

```diff
@@ -283,7 +283,7 @@ def show
     background_id = get_background_id(@sample)
     @report_page_params = { pipeline_version: @pipeline_version, background_id: background_id } if background_id
     @report_page_params[:scoring_model] = params[:scoring_model] if params[:scoring_model]
-    if @pipeline_run && (((@pipeline_run.adjusted_remaining_reads.to_i > 0 || @pipeline_run.results_finalized?) && !@pipeline_run.failed?) || @pipeline_run.report_ready?)
+    if @pipeline_run && @pipeline_run.report_ready?
```

Comment:
It would need to be tested for samples that have / have not completed. Also old samples that were run with different code. Since it's not required for this incarnation of the PR, I would leave it as is and deal with it in a PR dedicated to the ReportTable

Reply:
Sounds great :)